flink-core/src/main/java/org/apache/flink/configuration/ClusterOptions.java

```diff
@@ -65,11 +65,12 @@
 
 	@Documentation.Section(Documentation.Sections.EXPERT_FAULT_TOLERANCE)
 	public static final ConfigOption<Integer> CLUSTER_IO_EXECUTOR_POOL_SIZE = ConfigOptions
-		.key("cluster.io-executor.pool-size")
```

Comment:
add a deprecated key; the option already existed in previous versions.

Reply:
True. What about backporting it to 1.10.2 as well? I think this option has not been released yet.